.changeset/modern-colts-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-azure': patch
+---
+
+Add branch filter support
```

Comment:
Can you add a link to the `discovery` documentation here? Just to this: https://backstage.io/docs/integrations/azure/discovery

The changesets are used as the release notes later so having more information makes it easier for people when they upgrade

Reply:
Done.